src/Infrastructure/Service/Precondition/AbstractFileIteratingPrecondition.php

```diff
@@ -11,7 +11,7 @@
 use PhpTuf\ComposerStager\Infrastructure\Service\Finder\RecursiveFileFinderInterface;
 use PhpTuf\ComposerStager\Infrastructure\Value\PathList\PathList;
 
-abstract class AbstractLinkIteratingPrecondition extends AbstractPrecondition
+abstract class AbstractFileIteratingPrecondition extends AbstractPrecondition
```

Comment:
The semantic changes are all in this file. _Link_ iterating to _file_ iterating. Everything else is just updating the class and method names in their usages.

Reply:
Makes sense.